plugins/chat/app/services/chat/update_channel.rb

```diff
@@ -77,6 +78,10 @@ def mark_all_threads_as_read_if_needed(channel:, **)
       Jobs.enqueue(Jobs::Chat::MarkAllChannelThreadsRead, channel_id: channel.id)
     end
 
+    def update_site_settings_if_needed(channel:, **)
+      Jobs.enqueue(Jobs::Chat::UpdateChatThreadsEnabled)
```

Comment:
We need the service spec to ensure we enqueue this job and we need a spec for this job. Technically I would prefer jobs to have no logic but this is so simple here that I guess we can live with it

Reply:
Im fine inline, updating a channel is not a frequent action